pkg/execution/datasource/datasource_http_json.go

```diff
@@ -248,9 +260,17 @@ func (r *HttpJsonDataSource) Resolve(ctx context.Context, args ResolverArgs, out
 		httpMethod = http.MethodPatch
 	}
 
-	url := string(hostArg) + string(urlArg)
-	if !strings.HasPrefix(url, "https://") && !strings.HasPrefix(url, "http://") {
-		url = "https://" + url
+	rawURL := string(hostArg) + string(urlArg)
```

Comment:
what if add tiny helper

```go
func parseUrl(hostArg, urlArg []byte) (*url.URL, error) {
        rawURL := string(hostArg) + string(urlArg)
	return url.Parse(rawURL)
}
```

Reply:
i just modified you idea a bit, because i want to reuse the rawURL later for logging :+1: